virt/fusion_access_SAN/install-configure-fusion-access-san.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="install-configure-fusion-access-san"]
+= Installing and configuring {IBMFusionFirst}
+include::_attributes/common-attributes.adoc[]
+:context: install-configure-fusion-access-san
+
+toc::[]
+
+To use Red Hat {VirtProductName} with {IBMFusionFirst}, you must first install the {FusionSAN} Operator.
+
+Then you must create a Kubernetes pull secret and create the FusionAccess custom resource (CR).
+
+Finally, follow the Red Hat {product-title} web console wizard to configure the storage cluster, local disk, and file systems.
+
+include::modules/virt-installing-fusion-access-operator.adoc[leveloffset=+1]
+
+include::modules/virt-creating-pull-secret-fusion-san.adoc[leveloffset=+1]
+
+include::modules/virt-creating-fusionaccess-cr.adoc[leveloffset=+1]
+
+include::modules/virt-creating-storage-cluster-fusion-access-san.adoc[leveloffset=+1]
+
+include::modules/virt-creating-filesystem-fusion-access-san.adoc[leveloffset=+1]
+
+== Next steps
+
+Once you create a storage cluster with file systems, you can create a virtual machine (VM) on the storage cluster.
+
+Create a VM from an instance type or template and select one of the file systems you created as the storage type.
+
+For more information about creating VMs from instance types, see xref:../../virt/creating_vm/virt-creating-vms-from-instance-types.adoc#virt-creating-vms-from-instance-types[Creating virtual machines from instance types].
+
+For more information about creating VMs from templates, see xref:../../virt/creating_vm/virt-creating-vms-from-templates.adoc#virt-creating-vms-from-templates[Creating virtual machines from templates].
```

Comment:
If I were a target user, I would prefer two simple xrefs instead of all the lines in this section:

* https://92979--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/creating_vm/virt-creating-vms-from-instance-types

* https://92979--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/fusion_access_san/install-configure-fusion-access-san

Also, IMHO, the first sentence can be dropped because it should be in the prerequisites of these linked procedures.